docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
```

Comment:
It shows up first on my screen, I am talking about
docs/solutions/k8s-on-dcos/k8s-mesos_integration.png
but it sounds like we're talking about the same one.

I'm pretty sure someone from Mesosphere drew that diagram, or maybe @jdef ? It would be great if whoever has the original edit-able version of it could make the change I described above.